### ISSUE TITLE
instanceof error in page 

### ISSUE THREAD

**Issue by blairmacintyre** [servo/servo#23012]
While trying to run the demo at https://argonjs.github.io/three-web-layer/ on servo for mac, I got a bunch of these errors:

ERROR 2019-03-10T19:42:56Z: script::dom::bindings::error: Error at https://argonjs.github.io/three-web-layer/app.c61986b1.js:5:4695 invalid 'instanceof' operand d(...).SVGElement

@asajeffrey suggested I post an issue.

**Comment by jdm** [comment]
This is caused by us not implementing the SVGElement DOM interface (since we have no SVG support), so any code using `instanceof SVGElement` fails.

**Comment by blairmacintyre** [comment]
Right, thanks.  I think this is something deep in the HTML2Canvas package, which is checking if there's SVG Elements so it can handle them (even if there aren't SVG elements).

Not sure what to do about this.

**Comment by blairmacintyre** [comment]
Ok, for fun, we added 
```
window.SVGElement = class {}
window.SVGSVGElement = class {}
```
to the app, which stopped these crashes, but now html2canvas is throwing errors because `canvas2d.fillText` isn't implemented.

**Comment by jdm** [comment]
@blairmacintyre That link no longer exists; is there a better one to use?

**Comment by blairmacintyre** [comment]
https://www.aelatgt.org/three-web-layer/#    Moved to more appropriate org

**Comment by blairmacintyre** [comment]
@jdm wondered if this has an update.  FWIW, the current version of servo (downloaded just now from nightly builds on the mac) gives both the SVGElement error and 
`canvas::canvas_data] Unimplemented canvas2d.fillText. Values received: 1 MS (1-150), 6, 4, None.` errors

**Comment by jdm** [comment]
No update yet - there's a student just starting to work on some basic SVG support which may help eventually.

**Comment by blairmacintyre** [comment]
Is the fillText related?  We could remove SVGs from this example I think ...

**Comment by jdm** [comment]
As far as I know, fillText just causes a warning to appear in the terminal but shouldn't interrupt JS execution.
